File: scala/src/main/scala/com/madhukaraphatak/scala/simple/Expression.scala
package com.madhukaraphatak.scala.simple

object Expression {

  def main(args: Array[String]) = {
   val a = 10
   val isEven = if(a % 2 == 0) true else false

   println(s"$a is even $isEven")

    val blockResultExpression =  {
      1+2;
    };

   println(s"the block expression result is $blockResultExpression")
  }

}

File: rust/simple/src/bin/expression.rs
fn main() {
    let a = 10;

    let is_even = if a % 2 == 0 {true} else {false};
    println!("{} is even {}",a,is_even);    


    let block_result_statement =  {
      1+2;
    };

    println!("block statement result is {:?}",block_result_statement);
    
    let block_result_expression =  {
      1+2
    };

    println!("block expression result is {:?}",block_result_expression);
 
}
